jobs/openshift/openshift-ansible/build.groovy

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env groovy
+
+library "github.com/stevekuznetsov/release-library@skuznets/build-pipeline"
+
+buildPipeline(
+  "build",
+  cloneStep("openshift-ansible:test-base"),
+  [buildStep(
+    "build-rpms",
+    "src", "rpms",
+    "RUN umask 0002 && mkdir -p /srv/openshift-ansible/noarch && tito tag --offline --accept-auto-changelog && tito build --output=/srv/ --rpm --test --offline --quiet && createrepo /srv/openshift-ansible/noarch"
```

Comment:
can we push this into the repo or is it short enough?

Reply:
seems short enough to me though I don't like everything in a single line... you could split it up in a multi-line literal